e2e/remote/remote.go

```diff
@@ -40,13 +38,15 @@ func (c *ctx) remoteAdd(t *testing.T) {
 	}
 
 	for _, tt := range testPass {
-		argv := []string{"remote", "--config", config.Name(), "add"}
+		argv := []string{"--config", config.Name(), "add"}
 		argv = append(argv, tt.remote, tt.uri)
```

Comment:
Why is there an append to `argv` here (and other locations in the code)? Could you add it to the array on creation instead?

ie: `argv := []string{"--config", config.Name(), "add", tt.remote, tt.uri}`

Reply:
Fixed :+1: